features/events/traps/src/main/java/org/opennms/netmgt/trapd/TrapListener.java

```diff
@@ -120,25 +137,38 @@ public void run() {
     }
 
     public void stop() {
+        try {
+            m_twinSubscription.close();
+        } catch (IOException e) {
+            throw Throwables.propagate(e);
```

Comment:
This is getting deprecated https://github.com/google/guava/wiki/Why-we-deprecated-Throwables.propagate
we should probably not start using it now.